Code/Editor/TrackView/TrackViewSequence.cpp

```diff
@@ -732,7 +776,12 @@ void CTrackViewSequence::DeleteSelectedNodes()
 
 void CTrackViewSequence::SelectSelectedNodesInViewport()
 {
-    AZ_Assert(CUndo::IsRecording(), "Undo is not recording");
+    AZ_Assert(CUndo::IsRecording(), "Undo is not recording"); // Undo/Redo is managed in CTrackViewDopeSheetBase
```

Comment:
This is the old undo system, you sure you wanna check it ? Equivalent for the new one is AzToolsFramework::UndoRedoOperationInProgress()